script/upgrades/PoolRestructuring/TradingLimitsCfg.sol

```diff
@@ -0,0 +1,365 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// solhint-disable func-name-mixedcase, contract-name-camelcase, function-max-lines, var-name-mixedcase
+pragma solidity ^0.5.13;
+pragma experimental ABIEncoderV2;
+
+import { GovernanceScript } from "script/utils/Script.sol";
+
+import { Config } from "script/utils/Config.sol";
+import { CfgHelper } from "script/upgrades/PoolRestructuring/CfgHelper.sol";
+
+contract TradingLimitsCfg is GovernanceScript {
+  CfgHelper private cfgHelper;
+
+  struct Override {
+    address asset0;
+    address asset1;
+    address referenceRateFeedID;
+    Config.TradingLimit asset0Config;
+    Config.TradingLimit asset1Config;
+  }
+
+  constructor(CfgHelper _cfgHelper) public {
+    cfgHelper = _cfgHelper;
+  }
+
+  function tradingLimitsOverrides() public view returns (Override[] memory) {
+    Override[] memory overrides = new Override[](13);
+
+    // cEUR/axlEUROC
+    overrides[0] = Override({
+      asset0: cfgHelper.cEURProxy(),
+      asset1: cfgHelper.axlEUROCProxy(),
+      referenceRateFeedID: toRateFeedId("EUROCEUR"),
+      asset0Config: Config.TradingLimit({
+        enabled0: true,
+        timeStep0: 5 minutes,
+        limit0: 100_000,
+        enabled1: true,
+        timeStep1: 1 days,
+        limit1: 500_000,
+        enabledGlobal: false,
+        limitGlobal: 0
+      }),
+      asset1Config: Config.emptyTradingLimitConfig()
+    });
+
+    // cUSD/nativeUSDT
+    overrides[1] = Override({
+      asset0: cfgHelper.cUSDProxy(),
+      asset1: cfgHelper.nativeUSDTProxy(),
+      referenceRateFeedID: toRateFeedId("USDTUSD"),
+      asset0Config: Config.TradingLimit({
+        enabled0: true,
+        timeStep0: 5 minutes,
+        limit0: 2_500_000,
```

Comment:
I'm also curious why we're decreasing this one? I believe initially we wanted to support the same throughput as the combined of cUSD <-> axlUSDC/USDC.

Reply:
discussed offline